### PR TITLE
Remove our Run Above/Below commands

### PR DESCRIPTION
**Pull request by joyceerhl**
For https://github.com/microsoft/vscode-jupyter/issues/6025

VS Code has implemented this in core. It's controlled by a setting and disabled by default till Tuesday. This must only be merged on Tuesday.

### CODE REVIEW

**Review by DonJayamanne — APPROVED** [review]

**Review by IanMatthewHuff — APPROVED** [review]

**Comment by joyceerhl** [comment]
Should we migrate any keybindings that users have customized for our commands since we are deleting them?

**Comment by IanMatthewHuff** [comment]
> Should we migrate any keybindings that users have customized for our commands since we are deleting them?

I guess I think this might be more trouble than it's worth. This is native only commands right? For insiders customers it doesn't seem too bad to just have them update. My 2 cents at least. Avoid putting more code in.

File: news/1 Enhancements/6025.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Replace 'Run cells above' and 'Run cell and below' commands and cell toolbar buttons with VS Code's built-in 'Execute Above Cells' and 'Execute Cell And Below' commands and unified run button.
```
Comment (rchiodo) [inline news/1 Enhancements/6025.md:1]: I'm wondering if 'interrupt' is going to work correctly or not. We used to be able to cancel all the queued cells for run all when interrrupting. I'm guessing that doesn't work anymore.

Comment (rchiodo) [reply news/1 Enhancements/6025.md:1]: We probably will need some special logic in the executeCell commands to handle it.

**Review by rchiodo — APPROVED** [review]

**Comment by joyceerhl** [comment]
> This is native only commands right?

@IanMatthewHuff yes, but I believe these commands may be available to stable users in native notebooks as well.

**Pull request merged**